README.md

```diff
@@ -25,3 +25,22 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+For each itteration of my code, n/3, but as there are 3 subdivisions, we get
+T(n)=3T(n/3), except when only 2 subdivision can be made, in which circumstance
+we get T(n)=2T(n/2). Additionally my code does some checking/calculation each
+itterations, adding O(1). Not accounting for constant factors, this gives a
+final T(n)=T(n)=θ(n)
```

Comment:
Please use the notation we've used in class to denote membership in an asymptotic complexity class.